feat: Homepage

ajit-sharmaGH opened the pull request:
![image](https://user-images.githubusercontent.com/80311905/159071093-e058dc0f-58e0-461f-a507-c7162e7180cc.png)
![image](https://user-images.githubusercontent.com/80311905/159071202-89aa4c92-9c97-4bc5-ac10-e38978e92237.png)

ajit-sharmaGH committed:
feat: added navbar
  -made navbar responsive
  -added sidebar navigation menu
  -added badges on cart and wishlist icons

ajit-sharmaGH committed:
feat: added home page
-added banner
-added category option
-added services

SaurabhChawda reviewed (approved):
Give some space between heading and category and increase heading size because heading and subheading are looking the same to me.

ajit-sharmaGH committed:
reviewed: fixed the reviewed suggestion
-solved the sidebar responsiveness issue
-solved the conditional classNames for styling
-changed the color of category filter
-broke the code into different parts

ajit-sharmaGH commented:
> Give some space between heading and category and increase heading size because heading and subheading are looking the same to me.

Done

ajit-sharmaGH committed:
fixed search bar width

ajit-sharmaGH committed:
setup max width
